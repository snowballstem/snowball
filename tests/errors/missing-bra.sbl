strings
externals (stem)
backwardmode
define stem as (
  among
)
