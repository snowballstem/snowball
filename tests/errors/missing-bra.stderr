errors/missing-bra.sbl:2: ( omitted after strings
errors/missing-bra.sbl:4: ( omitted after backwardmode
errors/missing-bra.sbl:6: ( omitted after among
